ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs

```diff
@@ -198,6 +212,66 @@ public void SetFontStyle(string fontStyleString)
         }
 
         /// <summary>
+        /// Sets the text alignment.
+        /// </summary>
+        /// <param name="textAlign">The text alignment string.</param>
+        [ReactProp(ViewProps.TextAlign)]
+        public void SetTextAlign(string textAlign)
+        {
+            if (textAlign != null)
+            {
+                if (textAlign.Equals("auto"))
+                {
+                    _textAlignment = TextAlignment.DetectFromContent;
+                }
+                else
+                {
+                    _textAlignment = EnumHelpers.Parse<TextAlignment>(textAlign);
+                }
+                MarkUpdated();
+            }
+        }
+
+        /// <summary>
+        /// Sets the text decoration.
+        /// </summary>
+        /// <param name="textDecoration">The text decoration string.</param>
+        [ReactProp(ViewProps.TextDecorationLine)]
+        public void SetTextDecoration(string textDecoration)
+        {
+            if (textDecoration != null)
+            {
+                if (textDecoration.Contains("underline"))
+                {
```

Comment:
You'll need to account for the case where textDecoration is null, which will be to default back to false